### PR TITLE
Pypi package

### PR DESCRIPTION
**Pull request by ivanzvonkov**
Adds necessary code so repository can be deployed as pypi package

Currently, the package is available through
```bash
pip install -i https://test.pypi.org/simple/ cropharvest==0.0.9 
```

Once this PR is reviewed I'll publish a version on the official pypi repository.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
+        "xarray>=0.16.2",
+        "earthengine-api>=0.1.271",
+        "tqdm>=4.61.1",
+        "h5py>=3.1.0",
+        "rasterio>=1.2.6",
+        "openpyxl>=2.5.9",
+        "scikit-learn>=0.22.2",
     ],
+    python_requires=">=3.6",
+    include_package_data=True,
```
Comment (ivanzvonkov) [inline setup.py:31]: This line and the MANIFEST.in help the shapefiles be included in the package

File: setup.py [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
-        "openpyxl",
-        "sklearn",
+        "geopandas==0.9.0",
+        "xarray>=0.16.2",
+        "earthengine-api>=0.1.271",
+        "tqdm>=4.61.1",
+        "h5py>=3.1.0",
+        "rasterio>=1.2.6",
+        "openpyxl>=2.5.9",
+        "scikit-learn>=0.22.2",
```
Comment (ivanzvonkov) [inline setup.py:28]: Not specifying versions caused an infinite dependency resolution loop in Google Colab

File: cropharvest/__init__.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from cropharvest.columns import *
+from cropharvest.config import *
+from cropharvest.countries import *
+from cropharvest.crops import *
+from cropharvest.datasets import *
+from cropharvest.download import *
+from cropharvest.engineer import *
+from cropharvest.export import *
+from cropharvest.utils import *
```
Comment (ivanzvonkov) [inline cropharvest/__init__.py:9]: Helps the files from the top level cropharvest directory be included

**Comment by ivanzvonkov** [comment]
I was hoping to get a nice Google Colab notebook using the package but unfortunately, it's not possible due to this issue https://github.com/googlecolab/colabtools/issues/510. The untar for features times out because there are a large number of files in features.tar.gz.

File: README.md [hunk 4]
```diff
@@ -1,20 +1,12 @@
...
-CropHarvest is an open source remote sensing dataset for agriculture with benchmarks. It collects data a variety of agricultural land use datasets and remote sensing products.
+CropHarvest is an open source remote sensing dataset for agriculture with benchmarks. It collects data from a variety of agricultural land use datasets and remote sensing products.
 
 <img src="diagrams/labels_spatial_distribution.png" alt="Spatial distribution of labels" height="400px"/>
 
 ### Installation
 
-`cropharvest` can be pip installed by running `pip install -e .` where `.` is the path to the directory containing [`setup.py`](setup.py).
-
-For development, all CI tests are run using a virtual environment. The same can be set up locally using the following commands:
```
Comment (gabrieltseng) [inline README.md:11]: Might be good to keep the instructions, maybe under a "development" heading?

**Comment by gabrieltseng** [comment]
Thanks Ivan! LGTM

**Review by gabrieltseng — APPROVED** [review]

**Comment by gabrieltseng** [comment]
re: publishing `v1.0`, could you also wait until all the Canada stuff is processed? It should be done by the end of today

**Pull request merged**